apps/ledger-live-mobile/src/hooks/newsfeed/useIsNewsfeedAvailable.ts

```diff
@@ -0,0 +1,20 @@
+import useFeature from "@ledgerhq/live-common/featureFlags/useFeature";
+import {
+  cryptopanicAvailableRegions,
+  CryptopanicAvailableRegionsType,
+} from "./cryptopanicApi";
+import { useLocale } from "../../context/Locale";
+
+export function useIsNewsfeedAvailable() {
+  const { locale } = useLocale();
+  const newsfeedPageFeature = useFeature("newsfeedPage");
+
+  return (
+    newsfeedPageFeature?.enabled &&
+    newsfeedPageFeature?.params?.cryptopanicApiKey &&
+    cryptopanicAvailableRegions.includes(
+      locale as CryptopanicAvailableRegionsType,
+    ) &&
+    newsfeedPageFeature?.params?.whitelistedLocales?.includes(locale)
```

Comment:
isn't there already a `languages_whitelisted` parameter for this kind of stuff?

Reply:
and it's handled "natively" by useFeature() to return `enabled: false` in case a feature should be disabled for a given locale

Reply:
well to be exact: it's not handled directly by `useFeature`, it's implemented in the `getFeature` that is passed to the `FeatureFlagsProvider`

Reply:
Ah yes you are right, didn't know ... :face_exhaling: 
Don't know if i can switch this before code freeze...

Reply:
Fixed there : https://github.com/LedgerHQ/ledger-live/pull/2757 @ofreyssinet-ledger

Reply:
@ofreyssinet-ledger Fixed there https://github.com/LedgerHQ/ledger-live/pull/2758